model/src/cfd.rs

```diff
@@ -296,6 +298,35 @@ impl Order {
             self.opening_fee,
         )
     }
+
+    const OUTDATED_AFTER_MINS: i64 = 10;
```

Comment:
🔧 Might be worth documenting this constant too!

Reply:
I added a doccomment, not sure how much it helps. 
I would question the existence of this constant - I created it so the test reads better. Not sure if it's worth that...

Reply:
I like that it exists and that you documented it. But that's just my opinion haha.